nservicebus/testing/index.md

```diff
@@ -51,6 +51,8 @@ In Versions 5 and below, if [unobtrusive mode](/nservicebus/messaging/unobtrusiv
 
 snippet:SetupConventionsForUnitTests
 
+Note: With Version 6, unobtrusive message conventions configuration is no longer required for the NServiceBus Testing Framework.
```

Comment:
@SimonCropp does that require a comma? e.g. "In Versions 6 and above, unobtrusive message conventions configuration is no longer required for the NServiceBus Testing Framework."?